transports/azure-storage-queues/native-message-access_supported-versions_asqn_[10,11).partial.md

```diff
@@ -0,0 +1,7 @@
+## Access to the native Azure Storage Queue incoming message
+
+Accessing the native azure storage queue incoming message from behaviors and handlers can be beneficial for customizations and recoverability.  When a message is received, the transport includes the native `QueueMessage` in the message processing context. You can use the following code to retrieve the message details from a pipeline behavior:
+
+snippet: access-native-incoming-message
+
+The behavior above utilizes the native message’s `NextVisibleOn` property to identify when the message lost its lock due to aggressive prefetching and slow processing. If needed, a [custom recoverability policy](/nservicebus/recoverability/custom-recoverability-policy.md) can be implemented to bypass retry attempts that would inevitably fail because of the lost lock.
```

Comment:
Now that I'm thinking about this, I think I should add some information that the `QueueMessage` is read only and link to the Microsoft page

Reply:
I don't think we need to go overboard documenting all the angles.